src/server/cli.ts

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright 2017-2022 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { InvalidArgumentError, Option, program } from "commander";
+import { VERSION } from "./version";
+
+function assertIntegerOption(value: string): number {
```

Comment:
Do we have utils for integers? The logic looks generic.

Reply:
All utils will be moved to some /cli directory. This is not really generic because we need to throw very specific Exception. But sure, I will try to extract it into meaningful module.